src/set_theory/ordinal/basic.lean

```diff
@@ -863,9 +872,6 @@ by simp only [lt_iff_le_not_le, lift_le]
 quotient.sound ⟨(rel_iso.preimage equiv.ulift _).trans
  ⟨equiv_of_is_empty  _ _, λ a b, iff.rfl⟩⟩
 
-theorem zero_eq_lift_type_empty : 0 = lift.{u} (@type empty empty_relation _) :=
-by rw [← zero_eq_type_empty, lift_zero]
-
```

Comment:
Can you explain why this is redundant in the PR description?

Reply:
This is redundant because of `type_eq_zero_of_empty` and `lift_zero`.